linera-views/src/dynamo_db.rs

```diff
@@ -1588,7 +1602,7 @@ pub async fn create_dynamo_db_test_client() -> DynamoDbClient {
 }
 
 /// Helper function to list the names of tables registered on DynamoDB.
-pub async fn list_tables(
+pub async fn list_tables_from_client(
     client: &aws_sdk_dynamodb::Client,
```

Comment:
Perhaps a bit overkill (because it duplicates function signature) but technically, this one (and the similar ones) could be placed in a local extension trait for `aws_sdk_dynamodb::Client`.